src/MessagingThreadHistory/MessageMetadata.tsx

```diff
@@ -12,21 +12,25 @@ interface Props {
   className?: string;
   placement: "left" | "right" | "center";
   timestamp?: Date;
+  showReadStatus?: boolean;
   children: React.ReactNode;
 }
 
 export const MessageMetadata: React.FC<
   Props & { ref?: React.Ref<HTMLDivElement> }
 > = React.forwardRef((props: Props, ref: React.Ref<HTMLDivElement>) => {
-  const { className, placement, timestamp, children } = props;
+  const { className, placement, timestamp, showReadStatus, children } = props;
   return (
-    <div ref={ref} className={classNames(cssClass(`Message--${placement}`), className)}>
-      {children}
-      {timestamp && (
-        <span className={cssClass(`Timestamp--${placement}`)}>
-          {_formatDateForTimestamp(timestamp)}
-        </span>
-      )}
+    <div className={cssClass("Message--container")}>
+      <div ref={ref} className={classNames(cssClass(`Message--${placement}`), className)}>
+        {children}
+        {timestamp && (
+          <span className={cssClass(`Timestamp--${placement}`)}>
+            {_formatDateForTimestamp(timestamp)}
+          </span>
+        )}
+      </div>
+      {showReadStatus && <div className={cssClass("ReadReceipt")}>Read</div>}
```

Comment:
Just noting real quick that we'll want to make the specific string potentially customizable so that we can pass in translations, e.g. `readStatusText={t("Read")}`

Here's a good example of what this might look like! https://github.com/Clever/components/pull/550

And we'll want to pass in translated strings in the FamPo PR

Reply:
Just changed it to a consumer passed text field! Thanks for the reminder!